src/kernel/include/shared/cos_types.h

```diff
@@ -235,16 +236,17 @@ static inline unsigned long captbl_idsize(cap_t c)
  * 2GB-> = system physical memory
  */
 enum {
-	BOOT_CAPTBL_SRET       = 0,
-	BOOT_CAPTBL_SELF_CT    = 4,
-	BOOT_CAPTBL_SELF_PT    = 6,
-	BOOT_CAPTBL_SELF_COMP  = 8,
-	BOOT_CAPTBL_BOOTVM_PTE = 12,
-	BOOT_CAPTBL_PHYSM_PTE  = 14,
-	BOOT_CAPTBL_KM_PTE     = 16,
-
-	BOOT_CAPTBL_COMP0_CT           = 18,
-	BOOT_CAPTBL_COMP0_PT           = 20,
+	BOOT_CAPTBL_SRET          = 0,
+	BOOT_CAPTBL_SELF_CT       = 4,
+	BOOT_CAPTBL_SELF_PT       = 6,
+	BOOT_CAPTBL_SELF_COMP     = 8,
+	BOOT_CAPTBL_BOOTVM_PTE    = 12,
+	BOOT_CAPTBL_SELF_LOCAL_PT = 14,
```

Comment:
We definitely discussed this before. But before we put multiple page tables in the cos_info, which makes the code a little tedious, so want to simplify the code.
But now we have to use multiple mem_info. So again, I favor explicit separation between pgtbls for address translation and resource tracking.
And using multiple page tables consumes more memory, I do not think this is the issue we really care about now.
But if we use SELF_PT both for resource tracking and address translation, this will stress the address space resource of the manager.  Think about memory manager, if it managers 1G untyped memory, it just wastes lots of PTEs for the untyped memory, which limits the size of address space the manager can use. I probably miss something here.